app-misc/dateutils/dateutils-0.4.6.ebuild

```diff
@@ -5,11 +5,19 @@ EAPI=7
 
 DESCRIPTION="command line date and time utilities"
 HOMEPAGE="https://www.fresse.org/dateutils/"
-SRC_URI="https://bitbucket.org/hroptatyr/${PN}/downloads/${P}.tar.xz"
+
+case "${PV}" in
+	9999)
+		inherit git-r3
+		EGIT_REPO_URI="https://github.com/hroptatyr/${PN}.git"
+		;;
+	*)
+		SRC_URI="https://bitbucket.org/hroptatyr/${PN}/downloads/${P}.tar.xz"
```

Comment:
We usually don't [bother](https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D) with ${PN} in SRC_URI to make copy/pasting easier, even though it is valid syntax.